flight/mcu/lib/teensyShared/rk4.cpp

```diff
@@ -0,0 +1,90 @@
+#include "rk4.h"
+
+#include <Arduino.h>
+
+#include <array>
+#include <cmath>
+
+using std::array;
```

Comment:
what's this?

Reply:
This looks like a good idea that wasn't finished. 'using' is the new way to write typedefs in c++, and im guessing we wanted a typedef for std::array<float, 2> that is used a lot in this file. For now I'm just going to comment it out and revisit it during the semester.